Calibration/EcalCalibAlgos/test/EcalPhiSymReco_cfg.py

```diff
@@ -0,0 +1,72 @@
+import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
+
+# Cmd line options
+options = VarParsing ('analysis')
+options.register('produceByRun',
+                 True,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Accumulate EcalPhiSym RecHits by Run or by LuminosityBlock")
+options.register('saveFlatNano',
+                 True,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Produce FlatNanoAOD output")
+options.register('saveEDMNano',
+                 False,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Produce EDMNanoAOD output")
+options.register('saveEDM',
+                 False,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Produce stadard EDM output")
+options.inputFiles=[""]
+options.parseArguments()
+
+process = cms.Process("ECALPHISYM")
+process.load("FWCore.MessageService.MessageLogger_cfi")
+
+# Multi-threading
+process.options.numberOfStreams = cms.untracked.uint32(4)
+process.options.numberOfThreads = cms.untracked.uint32(4)
+
+# Conditions
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load('Configuration.Geometry.GeometryExtended2018Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2018Reco_cff')
+
+from CondCore.CondDB.CondDB_cfi import *
+process.GlobalTag = cms.ESSource("PoolDBESSource",
+                                 connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS'),
+                                 globaltag = cms.string('123X_dataRun2_v1')
```

Comment:
Better to use `'auto:run3_data'` so it's automaticcaly picked up from autoCond

Reply:
sure

Reply:
`auto:run3_data` does not work with Run2 data, some tags I'm accessing in the tests only have IOVs from 2021 onwards.
I can put `auto:run3_data` anyway if you prefer that

Reply:
My bad I misread your code, you should use `auto:run2_data`!

Reply:
done